### DISCUSSION TITLE
NextJS and PocketBase tutorial

### DISCUSSION THREAD

**Discussion by indrazm** [pocketbase/pocketbase#720] [Show and tell]
Hey There!
I just fall in love with PocketBase.
Thanks for making great product!

Just wanted to give back to the community.
I made some of tutorials for beginners.
I'll keep on updating this very soon!

Currently i am recording about file uploading and basic authentication.

https://www.youtube.com/watch?v=5ef8eWFLTT4&list=PLDG_B8e_ueVh44Z7WBf0py6gwnzE_SdTK

Many Thanks! :)

**Comment by andres-vizcaino** [comment]
Thanks bro, you helped me to deploy on cloud... do you know if i can deploy on Digital Ocean?... i have a free credit.

**Reply by ganigeorgiev** [reply]
@andres-vizcaino You could check this discussion - https://github.com/pocketbase/pocketbase/discussions/512#discussioncomment-3668130.

**Comment by greendesertsnow** [comment]
I couldn't see the authentication in your tutorial. Can you please cover that too?
